src/ierror.cpp

```diff
@@ -13,34 +17,35 @@ main( int argc, char **argv ) {
     signal( SIGPIPE, SIG_IGN );
 
     if ( argc != 2 ) {
-        printf( "Use -h for help\n" );
+        std::cout << "Use -h for help" << std::endl;
         return 1;
     }
 
-    if ( strcmp( argv[1], "-h" ) == 0 ) {
+    if (std::strcmp(argv[1], "-h") == 0) {
         usage( argv[0] );
         return 0;
     }
 
-    int errorCode = atoi( argv[1] );
+    int errorCode = std::atoi(argv[1]);
 
     if ( errorCode > 0 ) {
         errorCode = -errorCode;
     }
 
     char *mySubErrName = NULL;
     const char *myErrName = rodsErrorName( errorCode, &mySubErrName );
-    printf( "irods error: %d %s %s\n",
-            errorCode, myErrName, mySubErrName );
-    free( mySubErrName );
+
+    std::cout << "irods error: " << errorCode << ' ' << myErrName << ' ' << mySubErrName << std::endl;
+
+    std::free(mySubErrName);
     return 0;
 }
 
 
 void usage( char *prog ) {
-    printf( "Converts an irods error code to text.\n" );
-    printf( "Usage: %s [-vVh] errorNumber\n", prog );
-    printf( "The errorNumber can be preceeded with minus sign (-) or not\n" );
-    printf( " -h  this help\n" );
+    std::cout << "Converts an irods error code to text.\n"
+              << "Usage: " << prog << " [-vVh] errorNumber\n"
+              << "The errorNumber can be preceeded with minus sign (-) or not\n"
```

Comment:
As long as we're linking #433... :)

preceeded -> preceded

Reply:
Fixed